README.md

```diff
@@ -413,13 +413,15 @@ For fields in `participant` such as `actor` or `itemGrants`, server will return
 [Ref](https://developer.vainglorygame.com/docs#matches)
 
 - `.assets` - Array of [Asset](#assetModel)
-- `.createdAt`
-- `.duration`
-- `.gameMode`
-- `.patchVersion`
-- `.shardId`
-- `.stats`
-- `.titleId`
+- `.createdAt` - a string, the match timestamp, suitable for passing to JavaScript `Date` constructor
```

Comment:
This would be also good to highlight that it's the time on which the match was started, not when it ended